content/blog/_posts/inclusion-good-for-business.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Making Inclusion Part of Your Business is Better for Business. Period."
+date: 2021-05-14
+slug: inclusive-good-for-business
+author: Wendy Renstchler
+---
+
+As we mark the first Black History Month following the events of 2020, attention is being paid to how companies are responding to their learnings from the Black Lives Matter movement and putting them into practice. And more companies have recognized that one month a year doesn’t come close to being enough. The enormity of social justice inequalities coming to light last year has been a catalyst for many businesses to look within themselves at how they do business—as an organization, as an employer, and as a member of the community and the world.
+
+The recent _[Fortune/Deloitte CEO Study](https://www2.deloitte.com/us/en/pages/chief-executive-officer/articles/ceo-survey-gauging-priorities.html)_ found that 96 percent of CEOs now consider diversity, equity, and inclusion (that’s the DEI you’re starting to see more often in the corporate vernacular) to be a strategic priority, integral to building a culture that’s better reflective of the world in which we live. And it acknowledges and embraces the uniqueness that every single person brings to a company’s vision. Together, we are better, and that’s better for business.
+
+
+## Watch your language
+
+One way to do this from the outset is to recognize the [importance of language](https://www.bmc.com/blogs/2021-remote-work-considerations/) and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?
```

Comment:
One way to do this from the outset is to recognize the ~[importance of language](https://www.bmc.com/blogs/2021-remote-work-considerations/)~ importance of language and how we use it in speech and the written word. As the world has changed, and awareness along with it, once-standard terminology is being retired and replaced with more inclusive language. So, what do we mean by that?